room/compiler-processing-testing/src/main/java/androidx/room/compiler/processing/SyntheticKspProcessor.kt

```diff
@@ -55,7 +55,8 @@ class SyntheticKspProcessor private constructor(
             options,
             resolver,
             codeGenerator,
-            logger
+            logger,
+            XProcessingEnv.Language.JAVA
```

Comment:
we should probably pass this from somewhere into here instead of hardcoding. 

though after looking at the rest of the CL, you've not plugged it yet (to runProcessorTest). Were you planning to do that as a followup? It is probably better together so that there is a way to invoke it and then we can also write a runProcessorTest to ensure it is properly passed around.